File: Modul-3/soal-1/PRAK301_2410817210017_OrlandoSugian.py
inputan = []
while(len(inputan) < 3):
    x = input()
    x = x.split()
    inputan += x
    inputan = list(map(int,inputan))
inp1 = inputan[0]
inp2 = inputan[1]
inp3 = inputan[2]
if inp1 > inp3 and inp1 > inp2 and inp2 > inp3:
    print(f"{inp3} {inp2} {inp1}")
elif inp1 > inp2 and inp1 > inp3 and inp3 > inp2:
    print(f"{inp2} {inp3} {inp1}")
elif inp2 > inp3 and inp2 > inp1 and inp1 > inp3: 
    print(f"{inp3} {inp1} {inp2}")
elif inp2 > inp1 and inp2 > inp3 and inp3 > inp1:
    print(f"{inp1} {inp3} {inp2}")
elif inp3 > inp2 and inp3 > inp1 and inp1 > inp2:
    print(f"{inp2} {inp1} {inp3}")
else : 
    print(f"{inp1} {inp2} {inp3}")
    

File: Modul-3/soal-1/PRAK301_2410817210017_OrlandoSugian.c
#include<stdio.h>
int main()
{
    int inp1, inp2, inp3;
    scanf("%d %d %d", &inp1, &inp2, &inp3);

    if ( inp1 > inp2 && inp1 > inp3 && inp2 > inp3){
        printf("%d %d %d\n", inp3, inp2, inp1);
    }
    else if (inp1 > inp2 && inp1 > inp3 && inp3 > inp2){
        printf("%d %d %d\n", inp2, inp3, inp1);
    }
    else if(inp2 > inp1 && inp2 > inp3 && inp1 > inp3){
        printf("%d %d %d\n", inp3, inp1, inp2);
    }
    else if(inp2 > inp1 && inp2 > inp3 && inp3 > inp1){
        printf("%d %d %d\n", inp1, inp3, inp2);
    }
    else if(inp3 > inp1 && inp3 > inp2 && inp1 > inp2){
        printf("%d %d %d\n", inp2, inp1, inp3);
    } 
    else {
        printf("%d %d %d\n", inp1, inp2, inp3);
    }
    return 0;
}